.github/workflows/post_release_version_bump.yml

```diff
@@ -2,40 +2,29 @@ name: Automated Release Version Bumps
 
 on:
   release:
-    types: [published]   # run on release!
+    types: [published]
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "post_release_version_bump"
   post_release_version_bump:
-    # The type of runner that the job will run on
     runs-on: ubuntu-latest
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       - name: Clone repository
         uses: actions/checkout@v2
 
-      - name: Install Python
-        uses: actions/setup-python@v2
+      - name: Run post-release version bump
+        uses: PennyLaneAI/automation/version_bump_action@v1.0
         with:
-          python-version: '3.9'
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install pennylane
+          version_path: "./pennylane_qiskit/_version.py"
+          changelog_path: "./CHANGELOG.md"
+          release_status: "post_release"
 
-      - name: Run version bump
-        run: >
-          python .github/workflows/vb_script.py
-          --version_path "./pennylane_qiskit/_version.py"
-          --changelog_path "CHANGELOG.md" --post_release
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v3
         with:
           commit-message: post release version bump
           title: Version Bump
           body: updated changelog and _version.py
           branch: post-release-version-bump
-          reviewers: Jaybsoni
-          base: main
+          reviewers: rmoyard
```

Comment:
This will create a PR for both the pre-release and post-release version bumps with @rmoyard as the reviewer. If next release goes well, we can potentially just allow it merge directly.